### ISSUE TITLE
BioParallel errors: first remote error: cannot open the connection

### ISSUE THREAD

**Issue by MuyaoXi9271** [Bioconductor/BiocParallel#189]
Hi all experts,

I also meet the same error sometimes when I extract MS2 data by using "featureSpectra". Sometimes the error does not come out when I restart R or reboot the computer, but now this error keeps coming no matter whether I reboot the computer or restart R studio. I also try the solution of "register(SnowParam())". I have in total 20 samples.

Error: BiocParallel errors
2 remote errors, element index: 2, 6
3 unevaluated and other errors
first remote error: cannot open the connection
In addition: Warning messages:
1: In serialize(data, node$con) :
'package:stats' may not be available when loading
2: In serialize(data, node$con) :
'package:stats' may not be available when loading
3: In serialize(data, node$con) :
'package:stats' may not be available when loading
4: In serialize(data, node$con) :
'package:stats' may not be available when loading
5: In serialize(data, node$con) :
'package:stats' may not be available when loading
6: In serialize(data, node$con) :
'package:stats' may not be available when loading
7: stop worker failed:
wrong args for environment subassignment

Bests,
Muyao

**Comment by Jiefei-Wang** [comment]
Hello Muyao, 

Before we can look at your issue. Would you be able to post the version of `BiocParallel` as well as the session info? For example

```
> packageVersion("BiocParallel")
[1] ‘1.29.12’
> sessionInfo()
R version 4.1.2 (2021-11-01)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19044)
```

Best,
Jiefei

**Comment by MuyaoXi9271** [comment]
Hi Jiefei,

The problem has been solved in the closed issue #177 

Bests,
Muyao

**Comment by Jiefei-Wang** [comment]
Thanks for the feedback
